packages/babel-plugin-class-features/src/features.js

```diff
@@ -44,9 +44,17 @@ export function verifyUsedFeatures(path, file) {
       "@babel/plugin-class-features doesn't support decorators yet.",
     );
   }
-  if (hasFeature(file, FEATURES.privateMethods)) {
+
+  if (
+    path.isClassPrivateMethod() &&
+    !hasFeature(file, FEATURES.privateMethods)
+  ) {
+    throw path.buildCodeFrameError("Class private methods are not enabled.");
+  }
+
+  if (path.isClassPrivateMethod() && path.node.static) {
```

Comment:
We should also guard agains private accessors.

Reply:
Added!